sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/entries/CreateNewTravelEntrySteps.java

```diff
@@ -226,6 +259,58 @@ public CreateNewTravelEntrySteps(
               "User name is invalid");
           softly.assertAll();
         });
+
+    When(
+        "^I check if pick or create person popup in travel entry is displayed$",
+        () -> {
+          webDriverHelpers.waitUntilIdentifiedElementIsPresent(
+              PICK_OR_CREATE_PERSON_TITLE_DE); // wait for popup
+          String expectedTitle = "Person ausw\u00E4hlen oder erstellen";
+          String checkPopupTitle =
+              webDriverHelpers
+                  .getTextFromWebElement(PICK_OR_CREATE_PERSON_TITLE_DE)
+                  .toLowerCase(Locale.GERMAN);
+          softly.assertEquals(
+              checkPopupTitle,
+              expectedTitle.toLowerCase(Locale.GERMAN),
+              "Wrong popup title for Pick or create person");
+          softly.assertAll();
+        });
```

Comment:
not sure what to say about this method. Can we make it generic? to work for both DE and EN?

Reply:
I can't :( As you can see i need to check if German label is displayed
` String expectedTitle = "Person ausw\u00E4hlen oder erstellen";`